packages/uniforms/__tests__/connectField.tsx

```diff
@@ -184,4 +192,48 @@ describe('connectField', () => {
       },
     );
   });
+
+  describe('when rendered provides correct onChange', () => {
+    it('is defaults to field name', () => {
+      const Field = connectField(Test);
+      const value = 'some value';
+      const wrapper = mount(<Field name="another" />, reactContext);
+      wrapper.find(Test).prop('onChange')(value);
```

Comment:
Would it be possible to write these tests in a way that it would be easier to refactor to `react-testing-library` if needed in the future?

Reply:
I believe we cannot call `props` of a component in RTL, right? I _could_, however, rewrite it using stubs instead.

Reply:
Yes that's right, there is no access to the component props in RTL. If it is too time consuming to change these tests then I would say to not do it within this PR, but just wanted to mention this.